syncopy/specest/mtmfft.py

```diff
@@ -97,9 +98,13 @@ def mtmfft(data_arr, samplerate, nSamples=None, taper="hann", taper_opt=None):
 
     for taperIdx, win in enumerate(windows):
         win = np.tile(win, (nChannels, 1)).T
+        tapered = data_arr * win
+        # de-mean again after tapering - needed for Granger!
+        if demean_taper:
+            tapered = tapered - tapered.mean(axis=0)
         # real fft takes only 'half the energy'/positive frequencies,
         # multiply by 2 to correct for this
-        ftr[taperIdx] = 2 * np.fft.rfft(data_arr * win, n=nSamples, axis=0)
+        ftr[taperIdx] = 2 * np.fft.rfft(tapered, n=nSamples, axis=0)
         # normalization
```

Comment:
For memory-efficiency I'd suggest refactoring this a little:
```
win = np.tile(win, (nChannels, 1)).T
win *= data_arr
if demean_taper:
    win -= win.mean(axis=0)
ftr[taperIdx] = 2 * np.fft.rfft(win, n=nSamples, axis=0)
```

Reply:
Done it latest commit!